src/Containers/ProjectForm/ProjectForm.js

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from 'react';
+import { connect } from 'react-redux';
+import { addProject } from '../../actions/actions';
+import './ProjectForm.scss';
+
+const ProjectForm = (props) => {
```

Comment:
you can deconstruct the props right here ({ projects }) so that you don't have to do props.projects later on